api/handlers.go

```diff
@@ -80,15 +110,21 @@ func (app *Application) InsertArticle(w http.ResponseWriter, r *http.Request) {
 	_ = utility.ReadJSON(w, r, &article)
```

Comment:
error handling missing, if bad JSON body data pass then need to handle proper error with 400